docs/resources/aws_ssm_activations.md

```diff
@@ -0,0 +1,70 @@
+---
+title: About the aws_ssm_activations Resource
+platform: aws
+---
+
+# aws\_ssm\_activations
+
+Use the `aws_ssm_activations` InSpec audit resource to test properties of a collection of AWS SSM Activations.
+
+## Syntax
+
+ Ensure you have exactly 3 activations
+
+    describe aws_ssm_activations do
+      its('names.count') { should cmp 3 }
+    end
+    
+#### Parameters
+
+This resource does not expect any parameters.
+
+See also the [AWS documentation on SSM](https://docs.aws.amazon.com/systems-manager/?id=docs_gateway).
+
+## Properties
+
+|Property                     | Description|
+| ---                         | --- |
+|activation\_ids              | Provides  ID created by Systems Manager when you submitted the activation. |
+|created\_dates               | Provides the date the activation was created. |
+|default\_instance\_names     | Provides the name for the managed instance when it is created. |
+|descriptions                 | Provides a user defined description of the activation. |
+|expiration\_dates            | Provides the date when this activation can no longer be used to register managed instances. |
+|expired                      | Whether or not the activation is expired. |
+|iam\_roles                   | Provides the Amazon Identity and Access Management (IAM) role to assign to the managed instance. |
+|registration\_limits         | Provides the maximum number of managed instances that can be registered with this activation. |
+|registrations\_count         | Provides the number of managed instances already registered with this activation. |
```

Comment:
This should be plural.

Reply:
`registrations_count` seems ok for me - matches the AWS variable.

Reply:
going to mark this one as resolved as the existing variable appears to be fine.